tutorials/selenium/README.md

```diff
@@ -0,0 +1,14 @@
+# JxBrowser and Selenium WebDriver
+
+These projects demonstrate how to integrate a JxBrowser-based application with Selenium WebDriver.
+
+Creating of these example projects is described in the [tutorial](https://jxbrowser-support.teamdev.com/docs/tutorials/integration/selenium.html). 
+
+## Build and launch
+
+1. [Insert your license key](https://jxbrowser-support.teamdev.com/docs/guides/licensing.html#adding-the-license-to-a-project) into [the test application](test-application/src/main/java/Application.java).
```

Comment:
You can point to a line number using links in GitHub. Help the clients to find out the line where the key must be set.